load/PreValidateHandler.py

```diff
@@ -0,0 +1,47 @@
+# Handler.py
+# AWS Lambda Handler for PreValidate
```

Comment:
change this comment to:
This is a test stub intended to provide the same interface as the AWS Lambda Handler